README.md

```diff
@@ -67,48 +67,67 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
+###  Deployment without GitOps
 
-### ...without GitOps
+6. Create the secret in the 'rhdh-operator' namespace with the GitHub token:
 
-1.  Install the orchestrator Helm chart:
+   ```console
+   oc create secret generic backstage-backend-auth-secret --from-literal=BACKEND_SECRET=$BACKEND_SECRET --from-literal=GITHUB_TOKEN=$GITHUB_TOKEN -n rhdh-operator
+   ```
 
-    ```console
-    helm upgrade -i orchestrator orchestrator --set rhdhOperator.github.token=$GITHUB_TOKEN
-    ```
+7. Switch the context to the orchestrator namespace
 
-2.  Run the commands prompted at the end of the previous step to wait until the services are ready.
+   ```console
+   oc project orchestrator
+   ```
 
-### ... with GitOps
+8. Install the orchestrator Helm chart:
 
-1.  Install `Red Hat OpenShift Pipelines` and `Red Hat OpenShift GitOps` operators following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/gitops/README.md).
-    The Orchestrator installs RHDH and imports software templates designed for bootstrapping workflow development. These templates are crafted to ease the development lifecycle, including a Tekton pipeline to build workflow images and generate workflow K8s custom resources. Furthermore, ArgoCD is utilized to monitor any changes made to the workflow repository and to automatically trigger the Tekton pipelines as needed. This installation process ensures that all necessary Tekton and ArgoCD resources are provisioned within the same cluster.
+   ```console
+   helm upgrade -i orchestrator orchestrator -n orchestrator
+   ```
 
-2.  Run the following command to set up environment variables:
+   Run the commands prompted at the end of the previous step to wait until the services are ready.
 
-    ```console
-    ./hack/setenv.sh --use-default
-    ```
+### Deployment with GitOps
 
-    This script generates a `.env` file that contains all the calculated environment variables. You need to source the file to
-    run orchestrator installation command below by running:
+7.  Install `Red Hat OpenShift Pipelines` and `Red Hat OpenShift GitOps` operators following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/gitops/README.md).
+    The Orchestrator installs RHDH and imports software templates designed for bootstrapping workflow development. These templates are crafted to ease the development lifecycle, including a Tekton pipeline to build workflow images and generate workflow K8s custom resources. Furthermore, ArgoCD is utilized to monitor any changes made to the workflow repository and to automatically trigger the Tekton pipelines as needed. This installation process ensures that all necessary Tekton and ArgoCD resources are provisioned within the same cluster.
+
+8.  Run the following command to set up environment variables:
 
     ```console
-    source .env
+    ./hack/setup.sh --use-default
```

Comment:
added a comment on the other PR, but adding another one here not to miss it:
The name of the script is `configure_environment.sh` - should it be renamed to `setup.sh`?
https://github.com/parodos-dev/orchestrator-helm-chart/pull/167/files#diff-3fb28d1a89332aa8c52c027c08dc41bb2d05bd30319939a0bb000cd74dc5773e

Reply:
Yes, I was working on this PR after I pushed the changes to the `gh-pages` PR and it took me longer than expected to finish this one. The script shall remain `setup.sh`, I could not come up with a better name for a script that eventually generates a secret and labels namespaces.